docs/do-not-merge.md

```diff
@@ -182,163 +194,140 @@ Updated EARLIEST_BY and LATEST_BY function signatures as follows:
 | `segment/loadQueue/failed` |Number of segment assignments that failed to complete.|`dataSource`, `server`|0|
 
-[14453](https://github.com/apache/druid/pull/14453)
-[13197](https://github.com/apache/druid/pull/13197)
+[#13197](https://github.com/apache/druid/pull/13197)
 
 ### New metrics for task completion updates
 
-The following metrics have been added:
-* `task/status/queue/count`: Monitors the number of queued items
-* `task/status/updated/count`: Monitors the number of processed items
+| Metric | Description | Normal value |
+| ------ | ----------- | ---------- |
+| `task/status/queue/count` | Monitors the number of queued items | Varies |
+| `task/status/updated/count` | Monitors the number of processed items | Varies |
 
-[14533](https://github.com/apache/druid/pull/14533)
+[#14533](https://github.com/apache/druid/pull/14533)
 
 ### Added `groupId` to Overlord task metrics
 
 Added `groupId` to task metrics emitted by the Overlord. This is helpful for grouping metrics like task/run/time by a single task group, such as a single compaction task or a single MSQ query.
 
-[14402](https://github.com/apache/druid/pull/14402)
+[#14402](https://github.com/apache/druid/pull/14402)
 
 ### New metrics for monitoring sync status of `HttpServerInventoryView`
 
+TBD for name change
+
 |Metric|Description|Dimensions|Normal value|
 |------|-----------|-----------|----------|
-|`segment/serverview/sync/healthy`|Sync status of the Coordinator/Broker with a segment-loading server such as a Historical or Peon. Emitted by the Coordinator and Brokers only when [HTTP-based server view](../configuration/index.md#segment-management) is enabled. This metric can be used in conjunction with `segment/serverview/sync/unstableTime` to debug slow startup of the Coordinator.|`server`, `tier`|1 for fully synced servers, 0 otherwise|
-|`segment/serverview/sync/unstableTime`|Time in milliseconds for which the Coordinator/Broker has been failing to sync with a segment-loading server. Emitted by the Coordinator and Brokers only when [HTTP-based server view](../configuration/index.md#segment-management) is enabled.|`server`, `tier`|Not emitted for synced servers.|
-## Cluster management
-
-### Removed unused Coordinator dynamic configuration properties
+|`segment/serverview/sync/healthy`|Sync status of the Coordinator/Broker with a segment-loading server such as a Historical or Peon. Emitted by the Coordinator and Brokers only when HTTP-based server view is enabled. This metric can be used in conjunction with `segment/serverview/sync/unstableTime` to debug slow startup of the Coordinator.|`server`, `tier`|1 for fully synced servers, 0 otherwise|
+|`segment/serverview/sync/unstableTime`|Time in milliseconds for which the Coordinator/Broker has been failing to sync with a segment-loading server. Emitted by the Coordinator and Brokers only when HTTP-based server view is enabled.|`server`, `tier`|Not emitted for synced servers.|
 
```

Comment:
```suggestion
|`serverview/sync/healthy`|Sync status of the Coordinator/Broker with a segment-loading server such as a Historical or Peon. Emitted by the Coordinator and Brokers only when HTTP-based server view is enabled. This metric can be used in conjunction with `serverview/sync/unstableTime` to debug slow startup of the Coordinator.|`server`, `tier`|1 for fully synced servers, 0 otherwise|
|`serverview/sync/unstableTime`|Time in milliseconds for which the Coordinator/Broker has been failing to sync with a segment-loading server. Emitted by the Coordinator and Brokers only when HTTP-based server view is enabled.|`server`, `tier`|Not emitted for synced servers.|

```

Reply:
Manually updated in https://github.com/apache/druid/pull/14600/commits/0959b288c5a56b65cd0394f4d6f4be1b810a31ff